explorer/src/components/Consensus/Event/EventDetailsCard.tsx

```diff
@@ -63,13 +62,14 @@ export const EventDetailsCard: FC<Props> = ({ event }) => {
                   )}
                 </StyledListItem>
                 <StyledListItem title='Module'>{event.name.split('.')[0]}</StyledListItem>
-                <StyledListItem title='Call'>
+                {/* <StyledListItem title='Call'>
```

Comment:
Why did you remove (commented) name?

Reply:
No special reason, sometime as I was doing the changes needed, I was realizing the db was missing column or the indexer was not populating the data the way the FE needs, so I will be doing lap in between both and forgetting to uncomment breaking line after fixing the BE 🫠